### ISSUE TITLE
Party Vision

### ISSUE THREAD

**Issue by ejwright** [foundryvtt/dnd5e#3639]
I have created a party and have given all the players Ownership permissions, however when I drop the party onto the canvas no vision is enabled. If I swap for one of the tokens in the party it works fine.

**Comment by arbron** [comment]
Is vision enabled on the party's token?

**Comment by ejwright** [comment]
Yes, it is -- I have tried enabled and disabled, I even tried the different modes (e.g Darkvision)

**Comment by ejwright** [comment]
Just to follow-up, a little more experimenting showed that if I used the pre-created group included in Phandelver and Below (actor name 'Party'), it works fine after setting up vision and permission, still doesn't work on a group that I create myself
